Source/Serilog.Exceptions/Core/ExceptionEnricher.cs

```diff
@@ -13,49 +13,30 @@ namespace Serilog.Exceptions.Core
     /// </summary>
     public sealed class ExceptionEnricher : ILogEventEnricher
     {
-        public static readonly IExceptionDestructurer[] DefaultDestructurers =
-        {
-            new ExceptionDestructurer(),
-            new ArgumentExceptionDestructurer(),
-            new ArgumentOutOfRangeExceptionDestructurer(),
-            new AggregateExceptionDestructurer(),
-            new ReflectionTypeLoadExceptionDestructurer()
-        };
-
-        public static readonly IExceptionPropertyFilter IgnoreStackTraceAndTargetIdExceptionFilter =
-
-#if NET45
-            new IgnorePropertyByNameExceptionFilter(
-                nameof(Exception.StackTrace),
-                nameof(Exception.TargetSite));
-#else
-            new IgnorePropertyByNameExceptionFilter(
-                nameof(Exception.StackTrace));
-#endif
-
-        public static readonly IExceptionDestructurer ReflectionBasedDestructurer = new ReflectionBasedDestructurer();
+        public static readonly IEnumerable<IExceptionDestructurer> DefaultDestructurers = DestructuringOptionsBuilder.DefaultDestructurers;
```

Comment:
This can go now can't it?

Reply:
I left it for backward compatibility. After your comment, I marked it as `Obsolete`, to explicitly document intent.